### ISSUE TITLE
boidcmd setup: There was an error on setup

### ISSUE THREAD

**Issue by whoo** [Boid-John/boidcmd#10]
Hi,

There is an error when starting a job on cmdline.
- no wcgid was given
- can we add more debug option to solve that.

Thanks

#boidcmd setup
{ error: 'There was an error' }

**Comment by ghobson2013** [comment]
Hi thank you for reporting this issue, can you provide more info ... 

What linux distribution / OS version were you attempting this on.
Were you attempting to register a new device ? , was boinc-client installed and running prior to running boidcmd setup command ?
I will review the code to see in what instance you would get missing wcgid.
I would have expected more info on the screen at that time.

**Comment by ghobson2013** [comment]
all the setup command does is call boinccmd
```
boinccmd --project_attach http://www.worldcommunitygrid.org/ 1061556_a0c611b081f8692b7ef0c11d39e6105c
```
if this fails with the error you saw then boinc isn't correctly installed on your system.

**Comment by whoo** [comment]
Thanks for your answser.
I fix it but running boidcmd setup | in the right folder (writable)
(no information are push to app.boid.com | only name)

**Comment by ghobson2013** [comment]
yes that is normal , the Boid power is computed later and shows up within 24-48hrs on the website. if you see the device name your registered.

**Comment by ghobson2013** [comment]
Ok so we can close this ticket, this was a file permissions issue.
